<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/startbutton.png);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55555</width>
      <height>55555</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/background.jpg);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlelabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/title.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/exit.png);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="helplabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/help.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/continue.png);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <zorder>backgroundlabel</zorder>
   <zorder>StartButton</zorder>
   <zorder>titlelabel</zorder>
   <zorder>ExitButton</zorder>
   <zorder>helplabel</zorder>
   <zorder>continueButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
